README.md

```diff
@@ -10,13 +21,7 @@ It includes the following pipelines:
 3. [Stable Diffuison (with LoRA) C++ image generation pipeline](./image_generation/stable_diffusion_1_5/cpp/)
 4. [Latent Consistency Model (with LoRA) C++ image generation pipeline](./image_generation/lcm_dreamshaper_v7/cpp/)
 
-### License
```

Comment:
Please, update the link to Text generation C++ samples. They are in `samples` now. In addition, Python samples will be there as well.

Reply:
Links added

Reply:
Added links to python samples as well